scripts/similarity_query.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import sys
+
+from bugbug import bugzilla
+from bugbug.similarity import (
+    LSISimilarity,
+    NeighborsSimilarity,
+    Word2VecSoftCosSimilarity,
+    Word2VecWmdRelaxSimilarity,
+    Word2VecWmdSimilarity,
+)
+
+algo_to_model = {
```

Comment:
Define this in `similarity`, so you can re-use it in both scripts.

Reply:
And in the evaluation script

Reply:
Don't forget to also use it in the evaluation script (scripts/evaluate_similarity.py).